src/msspreadsheets/spreadsheet_constants.bal

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//HTTP Status Codes
+const int HTTP_STATUS_OK = 200;
+const int HTTP_STATUS_CREATED = 201;
+const int HTTP_STATUS_NO_CONTENT = 204;
```

Comment:
We can use status codes from the `ballerina/http` module instead of these constants

Reply:
Ok, Used status codes from the ballerina/http module and updated the code